README.md

```diff
@@ -1,97 +1,174 @@
-<!-- AUTO-GENERATED-CONTENT:START (STARTER) -->
-<p align="center">
-  <a href="https://www.gatsbyjs.org">
-    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
-  </a>
-</p>
-<h1 align="center">
-  Gatsby's default starter
-</h1>
+# PixelPoint Gatsby Starter
 
-Kick off your project with this default boilerplate. This starter ships with the main Gatsby configuration files you might need to get up and running blazing fast with the blazing fast app generator for React.
+## Table of Contents
 
-_Have another more specific idea? You may want to check out our vibrant collection of [official and community-created starters](https://www.gatsbyjs.org/docs/gatsby-starters/)._
+- [Getting Started](#getting-started)
+- [Usage](#usage)
+  - [Run website](#run-the-website)
+  - [Build website](#build-the-website)
+  - [Run built website](#run-built-website)
+  - [Clean Gatsby cache](#clean-gatsby-cache)
+- [Project Structure](#project-structure)
+- [Code Style](#code-style)
+  - [ESLint](#eslint)
+  - [Prettier](#prettier)
+  - [VS Code](#vs-code)
+- [Style Variables](#style-variables)
+
+## Getting Started
+
+1. Clone this repository
+
+```bash
+git clone git@github.com:pixel-point/gatsby-starter.git
+```
 
-## 🚀 Quick start
+2. Install dependencies
 
-1.  **Create a Gatsby site.**
+```bash
+npm install
+```
 
-    Use the Gatsby CLI to create a new site, specifying the default starter.
+3. Copy .env.example and rename it into .env
```

Comment:
Copy .env.example and rename it into .env. You can also fill there all remaining env variables required for the project.

Maybe we can extend it this way

Reply:
Good idea but our current env variables do not require any change after copying .env.example